Description: configure.in: stop checking for pre-C89
 No longer required, and no longer supported by recent automake.
Author: Adrian Bunk <bunk@debian.org>

--- agg-2.5+dfsg1.orig/configure.ac
+++ agg-2.5+dfsg1/configure.ac
@@ -8,10 +8,6 @@ dnl Checks for programs.
 AC_PROG_CC
 AC_PROG_CXX
 AC_ISC_POSIX
-AM_C_PROTOTYPES
-if test "x$U" != "x"; then
-  AC_MSG_ERROR(Compiler not ANSI compliant)
-fi
 AM_PROG_LIBTOOL
 AC_PROG_INSTALL
 
